docs/quick-start.md

```diff
@@ -0,0 +1,100 @@
+# Quick-start
+
+This guide walks you through the deployment of a TDP server, based on [`tdp-collection`](https://github.com/TOSIT-IO/tdp-collection/). For testing purpose, we'll use [SQLite](https://www.sqlite.org/index.html), a single-file database, and no authentication.
+
+For a more feature-rich testing environment, please use the provided [`docker-compose.yml`](../dev/).
+
+## Requirements
+
+- Installing [Python 3.7](https://www.python.org/downloads/release/python-3714/), using [pyenv](https://github.com/pyenv/pyenv) for example.
+- Installing [Poetry](https://python-poetry.org/) to manage Python dependencies.
+- Installing [SQLite3](https://www.sqlite.org/index.html).
+
+## Installation
+
+1. Clone the `tdp-server`, `tdp-collection` and `tdp-getting-started` repositories:
+   ```bash
+   git clone https://github.com/TOSIT-IO/tdp-server.git
+   git clone https://github.com/TOSIT-IO/tdp-collection.git
+   git clone https://github.com/TOSIT-IO/tdp-getting-started.git
+   ```
+1. Create the following directories:
+
+   - `server-deploy`, to hold the `tdp_vars` and the SQLite database.
+   - `tdp_vars`, to configure and track the history of the `tdp_vars`
+
+   ```bash
+   mkdir -p server-deploy/tdp_vars
+   ```
+1. Add the path of the `tdp_vars` directory in the `Inventory` of the `tdp-getting-started/ansible.cfg` file.
+
+   For example, using `vim tdp-getting-started/ansible.cfg`:
+
+   ```toml
+   # tdp-getting-started/ansible.cfg
+   [defaults]
+   # ...
+   inventory = inventory/hosts,inventory/tdp_vars,../server-deploy/tdp_vars
```

Comment:
```
   inventory = inventory/hosts,inventory/tdp_vars,../server-deploy/tdp_vars
```
Why keep `inventory/tdp_vars` and `../server-deploy/tdp_vars`. I'm not sure we would like this behavior

Reply:
You're right. Would this be ok?

```suggestion
   inventory = inventory/hosts,../server-deploy/tdp_vars
```

Reply:
Yes